### ISSUE TITLE
channels wont load 

### ISSUE THREAD

**Issue by NamedKitten** [omarroth/invidious#992]
error is 
2020-01-19 12:34:52 UTC 500 GET /channel/UCI4I6ldZ0jWe7vXpUVeVcpg 631.82ms
2020-01-19 12:34:54 UTC 200 GET /api/v1/auth/subscriptions?fields=authorId 53.71ms
Exception: Unexpected char '<' at 1:1 (JSON::ParseException)
  from /usr/lib/crystal/json/lexer.cr:371:7 in 'raise'
  from /usr/lib/crystal/json/lexer.cr:367:5 in 'unexpected_char'
  from /usr/lib/crystal/json/lexer.cr:366:11 in 'unexpected_char'
  from /usr/lib/crystal/json/lexer.cr:0:0 in 'next_token'
  from /usr/lib/crystal/json/parser.cr:13:12 in 'get_60_videos'
  from src/invidious.cr:3577:20 in '->'
  from lib/kemal/src/kemal/route.cr:255:3 in '->'
  from src/invidious/helpers/handlers.cr:255:3 in 'call'
  from /usr/lib/crystal/http/server/handler.cr:26:7 in 'call'
  from /usr/lib/crystal/http/server/handler.cr:26:7 in 'call_next'
  from lib/kemal/src/kemal/filter_handler.cr:21:7 in 'call'
  from /usr/lib/crystal/http/server/handler.cr:26:7 in 'call_next'
  from /usr/lib/crystal/http/server/handler.cr:26:7 in 'call_next'
  from /usr/lib/crystal/http/server/handler.cr:26:7 in 'call_next'
  from /usr/lib/crystal/http/server/handler.cr:26:7 in 'call_next'
  from /usr/lib/crystal/http/server/handler.cr:26:7 in 'call_next'
  from src/invidious/helpers/static_file_handler.cr:189:11 in 'call'
  from /usr/lib/crystal/http/server/handler.cr:26:7 in 'call'
  from /usr/lib/crystal/http/server/handler.cr:26:7 in 'call'
  from /usr/lib/crystal/http/server/handler.cr:26:7 in 'call_next'
  from lib/kemal/src/kemal/init_handler.cr:12:7 in 'process'
  from /usr/lib/crystal/fiber.cr:255:3 in 'run'
  from ???

**Comment by Perflyst** [comment]
Is your IP blocked by Google? Try curl via IPv4 and IPv6 and grep for captcha

**Comment by Perflyst** [comment]
Also duplicate of #963

**Comment by omarroth** [comment]
Closing in favor of #963.
